src/Entity/Timeline/Measure.php

```diff
@@ -129,6 +122,11 @@ class Measure
 
     private $savedThemes;
 
+    /**
+     * @Assert\Valid
+     */
+    protected $translations;
```

Comment:
I would name it `translatedTitles`.

Reply:
this translations are meant to hold all the translated field of this entity

also, as we are using the `A2lix\I18nDoctrineBundle\Doctrine\ORM\Util\Translation` trait, the property name should be `translations`.